resources/services/bigquery/bigquery_datasets.go

```diff
@@ -123,8 +123,8 @@ func fetchBigqueryDatasets(ctx context.Context, meta schema.ClientMeta, parent *
 		list, err := c.RetryingDo(ctx, call)
 		if err != nil {
 			if isAccessErrorToIgnore(err, c.ProjectId) {
-				return diag.FromError(err, diag.USER, diag.WithType(diag.ACCESS),
-					diag.WithDetails("Please verify the project id was configured correctly or BigQuery API is enabled in current project. Project names can't be used when fetching BigQuery resources"))
+				return diag.FromError(err, diag.RESOLVING, diag.WithSeverity(diag.IGNORE),
```

Comment:
why not keep it `diag.ACCESS`?

Reply:
This kind of error means the same as `SERVICE_DISABLED` error which is caught in `IgnoreErrorHandler` and reported as `RESOLVING` and `IGNORE`